src/EnergyPlus/HVACControllers.cc

```diff
@@ -572,35 +568,24 @@ void GetControllerInput(EnergyPlusData &state)
 
             ControllerProps(Num).ControllerName = AlphArray(1);
             ControllerProps(Num).ControllerType = CurrentModuleObject;
-            {
-                auto const SELECT_CASE_var(AlphArray(2));
-                if (SELECT_CASE_var == "TEMPERATURE") {
-                    ControllerProps(Num).ControlVar = HVACControllers::CtrlVarType::Temperature;
-                } else if (SELECT_CASE_var == "HUMIDITYRATIO") {
-                    ControllerProps(Num).ControlVar = HVACControllers::CtrlVarType::HumidityRatio;
-                } else if (SELECT_CASE_var == "TEMPERATUREANDHUMIDITYRATIO") {
-                    ControllerProps(Num).ControlVar = HVACControllers::CtrlVarType::TemperatureAndHumidityRatio;
-                    //        CASE ('FLOW')
-                    //          ControllerProps(Num)%ControlVar  = iFlow
-                } else {
-                    ShowSevereError(state, std::string{RoutineName} + CurrentModuleObject + "=\"" + AlphArray(1) + "\".");
-                    ShowSevereError(state,
-                                    "...Invalid " + cAlphaFields(2) + "=\"" + AlphArray(2) +
-                                        "\", must be Temperature, HumidityRatio, or TemperatureAndHumidityRatio.");
-                    ErrorsFound = true;
-                }
+
+            ControllerProps(Num).ControlVar =
```

Comment:
But of course when the SCV is doing string stuff, we convert it to getEnumerationValue.

Reply:
Excuse moi, monsieur, would you happen to have any Grey Poupon?